### DISCUSSION TITLE
Install OBD with RPM failed, no package found

### DISCUSSION THREAD

**Discussion by zhangysh1995** [oceanbase/oceanbase#68] [Q&A] [answered]
**Describe the bug (__must be provided__)**

Following the [doc ](https://open.oceanbase.com/docs/community/oceanbase-database/V3.1.0/use-obd-to-obtain-the-oceanbase-database) to install ODB, but failed:
````
$ sudo yum makecache
$ sudo yum install -y ob-deploy-1.0.0-1.el7.x86_64.rpm
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror-hk.koddos.net
 * centos-sclo-rh: mirror.hkt.cc
 * centos-sclo-sclo: mirror.hkt.cc
 * epel: mirror2.totbb.net
 * extras: mirror-hk.koddos.net
 * updates: mirror-hk.koddos.net
No package ob-deploy-1.0.0-1.el7.x86_64.rpm available.
Error: Nothing to do
````

**Your Environments (__must be provided__)**

* OS: `3.10.0-1160.el7.x86_64`
* Compliler: `gcc/g++ 4.8.5`
* CPU: `Intel(R) Xeon(R) CPU E5-2698 v4 @ 2.20GHz`
* Commit id (e.g. `a3ffc7d8`): `12676080774af2940e15911b9f2658f59c52899a`

**How To Reproduce(__must be provided__)**
As described.

**Expected behavior**

Install successfully.

**Additional context**
````
$ cat /etc/os-release 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
````

**Comment by zhangysh1995** [comment]
Additionally, the template asks for `OS: uname -a`, but I suppose `uname -r` is required?

**Comment by frf12** [comment]
You should get ob-deploy rpm package before excute `yum install -y ob-deploy-1.0.0-1.el7.x86_64.rpm`.
Use `wget https://mirrors.aliyun.com/oceanbase/community/stable/el/7/x86_64/ob-deploy-1.0.0-1.el7.x86_64.rpm` to get the ob-deploy rpm package.

**Comment by MizuhaHimuraki** [comment]
I think, you should add OceanBase Yum Repo to your yum configuration first. Please refer to this [doucumentation](https://github.com/oceanbase/obdeploy/blob/master/README-CN.md).

Btw, this documentation is available in the website too, see here
![image](https://user-images.githubusercontent.com/57518860/120315842-59982100-c30f-11eb-88a3-9eefb1309092.png)

**Comment by longdafeng** [comment, marked as answer]
please refer to  https://open.oceanbase.com/quickStart

Have you finished these steps?
```
sudo yum install -y yum-utils
sudo yum-config-manager --add-repo https://mirrors.aliyun.com/oceanbase/OceanBase.repo
sudo yum install -y ob-deploy
```
